extraTs/index.ts

```diff
@@ -0,0 +1,33 @@
+import { diamond, equilateral } from './patterns';
+import { permissions } from './constants';
+import { hasPermission, validateUsers } from './utils';
+diamond(3);
+equilateral(3);
+
+console.log(hasPermission(permissions.getUsers, 'trainee', 'read'));
+console.log(hasPermission(permissions.getUsers, 'trainee', 'write'));
+console.log(hasPermission(permissions.getUsers, 'trainer', 'read'));
+console.log(hasPermission(permissions.getUsers, 'trainee', 'all'));
+
+import { Iusers } from './interfaces';
```

Comment:
keep all the import statements at the top only

Reply:
Done